KUBERNETES.md

```diff
@@ -0,0 +1,231 @@
+## Running: Kubernetes
+
+This project uses [Lightbend's Platform Tooling](https://developer.lightbend.com/docs/reactive-platform-tooling/latest/) to
+simplify deployment to [Kubernetes](https://kubernetes.io/). Follow the steps below to install Online Auction in your
```

Comment:
"simplify deployment of the Lagom online auction to ....

Reply:
done